### ISSUE TITLE
Ensure correct student names are in the application confirmation emails

### ISSUE THREAD

**Issue by ramonh** [rails-girls-summer-of-code/rgsoc-teams#955]
As quickly gone over in PR #951, we've had to temporarily disable application confirmation emails as we had an incident where students were sent the opposite applications with the opposite names.

We need to have a way to ensure that the correct `student_index` is used for `student0` and `student1`.

**Comment by cesswairimu** [comment]
Hi @ramonh I would love to work on this issue

**Comment by ramonh** [comment]
Hey @cesswairimu, sure thing, thank you so much for offering to help!

Please make sure to let me know if I can help you in any way.

**Comment by cesswairimu** [comment]
@ramonh after rebasing I am getting this error. Tried to find some work around but nothing seems to work.
![screenshot from 2018-03-13 11-32-39](https://user-images.githubusercontent.com/17081074/37330570-9be48456-26b2-11e8-95db-9cf52147dd4e.png)

**Comment by ramonh** [comment]
Hey @cesswairimu Thanks for bringing this up!

I tried checking out your branch and oddly enough, it seems to be working on my end. However, I'm on macOS so I suspect it could be a gem installation issue.

Did I understand correctly that it worked before?

Maybe one thing you could try is reinstalling the gems in your bundle. You can do that by running the following:

    $ bundle install --force

**Comment by cesswairimu** [comment]
yes, It worked before. I forked the repo a while back and after pulling the lastest changes, I am  getting the error. Could be something to do with the upgrade to ruby-2.5.0 I am guessing. I just tried to fork the clone the repo directly and i am getting the same error.

**Comment by ramonh** [comment]
Very odd, and I take it re-installing the gems didn't work?

**Comment by cesswairimu** [comment]
Yep reinstalled successfully but the error persists

**Comment by ramonh** [comment]
Aw! 

Let's try a theory. Could you try opening the `.ruby-version` and changing it back to, what I believe was 2.3.1?

Then you can `cd` out and back in to try?

**Comment by cesswairimu** [comment]
:+1: That works  Thanks a lot
